fanduelOdds.php

```diff
@@ -35,13 +94,218 @@ private function getLiveMarketIds() {
          *
          * @return array Array of live market ids
          */
-    }
 
-    private function getMarketPrices() {
+        // setup request to ListMarketCatalogue API
+        $liveMarketsRequestPayload = json_encode([
+            "listMarketCatalogueRequestParams" => [
+                "marketFilter" => [
+                    "eventTypeIds" => [self::BASKETBALL_ID],
+                    "competitionIds" => [self::NBA_ID],
+                    "marketTypes" => self::MARKET_TYPES
+                ],
+                "maxResults" => 1000
+            ]
+        ]);
+        
+        $requestHeaders = [
```

Comment:
Extract CURL call to its own function so the code here is a call to that function.  This function should not know how the call is made (CURL, WGET or whatever - basically, decouple the HTTP call from the business logic)